### PR TITLE
Adding a CMake 2.8 formula

### PR DESCRIPTION
**Pull request by cloderic**
This provides a formula to build previous 2.8.X versions of CMake

### CODE REVIEW

**Comment by DomT4** [comment]
What's the rationale on this? Will certain things not compile with the newer CMake versions?

**Comment by cloderic** [comment]
Yes indeed, CMake 3.0 has dropped support for some of the 2.X features (http://www.cmake.org/cmake/help/v3.0/release/3.0.0.html#deprecated-and-removed-features)

**Comment by DomT4** [comment]
Alright. In that case can you address any comments ` brew audit --strict ` has locally and then rebase to squash your commits into one, Thanks! Ideal commit message would be ` cmake28 2.8.12.2 (new formula) `.

**Comment by cloderic** [comment]
Done !

File: cmake28.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+require 'formula'
```
Comment (DomT4) [inline cmake28.rb]: :scissors: this line

File: cmake28.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+class NoExpatFramework < Requirement
+  def expat_framework
+    '/Library/Frameworks/expat.framework'
```
Comment (DomT4) [inline cmake28.rb]: Double quotes everywhere throughout the formula.

File: cmake28.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  satisfy :build_env => false do
+    not File.exist? expat_framework
```
Comment (DomT4) [inline cmake28.rb]: ` !File.exist? expat_framework ` rather than ` not `.

File: cmake28.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      --system-libs
+      --no-system-libarchive
+      --datadir=/share/cmake
+      --docdir=/share/doc/cmake
+      --mandir=/share/man
+    ]
+
+    system "./bootstrap", *args
+    system "make"
+    system "make install"
```
Comment (DomT4) [inline cmake28.rb]: ` system "make", "install" `.

**Comment by DomT4** [comment]
Ping.

**Comment by cloderic** [comment]
I was busy preparing then participating to GDC I didn't found the time yet to take your remarks into account  I should have some time next week. 



—

**Comment by DomT4** [comment]
Thanks. No worries, Just ping stuff that's gone quiet for over a week to make sure people haven't lost interest/forgotten/etc :smile_cat:.

**Comment by DomT4** [comment]
Any update on this?

**Pull request closed without merging**